inductiva/molecules/scenarios/protein_solvation/README.md

```diff
@@ -0,0 +1,44 @@
+# Protein Solvation Scenario
+
+This scenario models the dynamics of a protein solvated in water inside a cubic box. The system is modelled with the motion equations for each individual atoms according to the prescription of a molecular force field approximation.
+
+To initilaize the scenario, the user can define the following parameters:
+-- Protein .pdb file: PDB file containing the structure of the protein to simulate;
```

Comment:
Shall we say that it can be "any" type of structure? Or just will be more restrictive?

Reply:
I think it is best to be just the protein.